script/proto_gen.sh

```diff
@@ -1,31 +1,34 @@
-apk update && apk add git
-
-DESTINATION_DIR=/mnt/proto/otel
+#!/usr/bin/env bash
+DESTINATION_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")/../proto/otel
+GPBMETA_DIR="GPBMetadata"
+OTEL_DIR="Opentelemetry"
 REPO_DIR=opentelemetry-proto
 
-mkdir -p $DESTINATION_DIR
-rm -R ./$REPO_DIR
-git clone https://github.com/open-telemetry/$REPO_DIR
+cd "${DESTINATION_DIR}" || exit
+rm -rf ./${REPO_DIR}
```

Comment:
probably want a check for something here.  If $`{REPO_DIR}` ends up being empty or being inadvertently set some other way......

Reply:
It's set a couple of lines up, so I _think_ this is safe. I've confirmed that it's portable enough to be run from / or /scripts and still work...